plugins/rbac/src/hooks/useRoles.ts

```diff
@@ -70,42 +77,79 @@ export const useRoles = (
     permission: policyEntityUpdatePermission,
     resourceRef: policyEntityUpdatePermission.resourceType,
   });
+
+  React.useEffect(() => {
+    const conditionsPromises: Promise<RoleWithConditionalPoliciesCount>[] =
+      (Array.isArray(roles) &&
+        roles.map(async role => {
+          const conditionalPolicies = await rbacApi.getRoleConditions(
+            role.name,
+          );
+
+          return {
+            ...role,
+            conditionalPoliciesCount: Array.isArray(conditionalPolicies)
+              ? conditionalPolicies.length
+              : 0,
+          };
+        })) ||
+      [];
+    const fetchAllPermissionPolicies = async () => {
+      if (Array.isArray(roles)) {
```

Comment:
Not required as you have already added this check here https://github.com/janus-idp/backstage-plugins/pull/1714/files#diff-de9e9b3435ef119eaf0550b2d4ff253ec247e8b76f386f0d5fadd8e111218aa7R109